lib/services/livesync/platform-livesync-service-base.ts

```diff
@@ -78,7 +79,7 @@ export abstract class PlatformLiveSyncServiceBase implements IPlatformLiveSyncSe
 			device.applicationManager.checkForApplicationUpdates().wait();
 
 			let appIdentifier = this.liveSyncData.appIdentifier;
-			if (!device.applicationManager.isApplicationInstalled(appIdentifier).wait()) {
+			if (!device.applicationManager.isApplicationInstalled(appIdentifier).wait() || !this.$platformService.isPlatformBuilt(deviceAppData.platform.toLowerCase()).wait()) {
 				this.$logger.warn(`The application with id "${appIdentifier}" is not installed on device with identifier ${device.deviceInfo.identifier}.`);
```

Comment:
The message indicates that the app is not installed, but in fact we can get here when the application is installed. Consider changing the message.